WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -551,6 +553,7 @@ static GSourceFuncs _handlerIntervention =
             Core::JSON::String CookieStorage;
             Core::JSON::String LocalStorage;
             Core::JSON::Boolean LocalStorageEnabled;
+            Core::JSON::String LocalStorageSize;
```

Comment:
Is there a reason why not one of the Core::JSON number types were used instead of a string? That way you would have the advantage of  using the standard parser validation and conversion

Reply:
@MFransen69 I modified the type as per your suggestion . Could you please check it.